.azure-pipelines/warden.yml

```diff
@@ -24,6 +24,34 @@ jobs:
       python setup.py sdist -d $(Build.ArtifactStagingDirectory) --format zip
     displayName: 'Build Package'
 
+  - task: CopyFiles@2
+    displayName: 'Copy files for github release'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)'
+      Contents: 'scripts/*/verify_agent_os.py'
+      TargetFolder: '$(Build.BinariesDirectory)/sdk-tools'
```

Comment:
`$(Build.BinariesDirectory)` for this purpose I think we should use `$(Build.ArtifactStagingDirectory)`

Reply:
We should also publish this as a build artifact.

Reply:
Looks like we already publish the the build artifacts so just be sure to put this sdk-tools under the artifactstagingdirectory.

Reply:
Looks like you still ended up using BinariesDirectory instead of ArtifactStagingDirectory. Is there a reason you stuck with that?